frontend/src/components/forms/ProspectUserForm.vue

```diff
@@ -0,0 +1,390 @@
+<script lang="ts">
+export default {
+  name: "ProspectUserForm",
+};
```

Comment:
The forms appearance in the ui needs cleaning as you know already. I think you can do another attempt here at just re-using the users form. But no worries if you can't figure it out .. i can do that but try one more time -- wait lol you did below well then delete this file

Reply:
We'll talk about this one tomorrow